pkg/cmd/server/server.go

```diff
@@ -345,6 +347,34 @@ func (c *Config) Complete(ctx context.Context) (RunnableServer, error) {
 	}, nil
 }
 
+func (c *Config) buildMiddleware(dispatcher dispatch.Dispatcher, ds datastore.Datastore) ([]grpc.UnaryServerInterceptor, []grpc.StreamServerInterceptor) {
+	finalUnaryMiddleware, finalStreamingMiddleware := DefaultMiddleware(log.Logger, c.GRPCAuthFunc, !c.DisableVersionResponse, dispatcher, ds)
+	if len(c.ReplaceUnaryMiddleware) != 0 {
+		finalUnaryMiddleware = c.ReplaceUnaryMiddleware
+	}
+	if len(c.ReplaceStreamingMiddleware) != 0 {
```

Comment:
So Replace goes first... but it doesn't actually replace any of the middleware besides the default?

Reply:
Correct, it's a replacement for the defaults. Maybe I should make that more explicit in the name, like `ReplaceDefaultUnaryMiddleware` and `ReplaceDefaultStreamingMiddleware?